security-analysis/security-analysis-api/src/main/java/com/powsybl/security/results/ThreeWindingsTransformerResult.java

```diff
@@ -12,10 +12,9 @@
 import java.util.Objects;
 
 /**
+ * Contains state variables of a three-windings transformer. The state variables supported are active and reactive power
```

Comment:
Similarly `Contains state variables of a three-winding transformer which is monitored during security analysis through a {@link StateMonitor}`?